packages/devextreme/js/__internal/ui/chat/chat_message_box.ts

```diff
@@ -70,10 +70,17 @@ class MessageBox extends DOMComponent<MessageBox, Properties> {
       .addClass(CHAT_MESSAGE_BOX_TEXTAREA_CLASS)
       .appendTo(this.element());
 
+    const onEnterKey = (e): void => {
+      if (e.event.shiftKey !== true) {
+        this._sendHandler(e);
+      }
+    };
+
     this._textArea = this._createComponent($textArea, TextArea, {
       activeStateEnabled,
       focusStateEnabled,
       hoverStateEnabled,
+      onEnterKey,
```

Comment:
I don't like the solution because add break line during sending looks ugly.
Moreover, when onMessageSend event is calling it gets text value with this line break. It means that text inside message bubble will with this line break
If I type 'd' and press enter I will get 'd\n'

Reply:
It will issue when Roman will support line breaks in messagebubbles.

Reply:
[commit](https://github.com/DevExpress/DevExtreme/pull/28003/commits/d6ca8e3ae9f19ab2e7efeee4eb8b7dc06dc7e7bd)